electron/gog/games.ts

```diff
@@ -45,11 +45,7 @@ const installedGamesStore = new Store({
 })
 
 function verifyProgress(stderr: string): boolean {
```

Comment:
Since this function will always return `true`, it means that the places we have a check for it, like line 169 and so on, should be refactored as well?